### DISCUSSION TITLE
post/my-personal-website-almost-done

### DISCUSSION THREAD

**Discussion by giscus** [zihan-ch/blog#1] [General]
# post/my-personal-website-almost-done

About how I made this website, what tech stacks or tools did I choose to build this website and what I am going to do next.

https://3000-zihanch-blog-rz15m6tcab3.ws-us70.gitpod.io/post/my-personal-website-almost-done

**Comment by arstnei0** [comment]
This comment is powered by [`Giscus`](https://giscus.app). It is really easy to configure a comment system powered by `Giscus`!🚀
